source/CMakeLists.txt

```diff
@@ -12,11 +12,22 @@ configure_file(
     @ONLY)
 
 include(GNUInstallDirs)
-add_library(${TARGET_LOADER_NAME}
-    SHARED
-    ""
-    ${CMAKE_CURRENT_BINARY_DIR}/ZeLoaderVersion.rc
-)
+if (DYNAMIC_LOAD_LOADER)
```

Comment:
Recommend the top level cmake option be named after the result rather than the behavior, like `BUILD_STATIC`

Reply:
good point, that would differentiate.

Reply:
Fixed in the latest patches.